spec/services/organization_update_service_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "rails_helper"
 
-describe OrganizationUpdateService, skip_seed: true do
+RSpec.describe OrganizationUpdateService, skip_seed: true do
```

Comment:
I have no clue what this `skip_seed` is for. It got added in this PR https://github.com/elasticspoon/human-essentials/commit/1fbb90570e5a1180ef74ee9f6522ff86e6e9a821 and I'm not sure about the reasoning.

Reply:
Might have been an early attempt at doing exactly what we're doing now. :smile: